crates/bitwarden-exporters/src/cxf/import.rs

```diff
@@ -17,7 +17,7 @@ pub(crate) fn parse_cxf(payload: String) -> Result<Vec<ImportingCipher>, CxfErro
     Ok(items)
 }
 
-/// Convert a CXF timestamp to a DateTime<Utc>.
+/// Convert a CXF timestamp to a `DateTime<Utc>`.
```

Comment:
Should we instead do a reference to the type?
```suggestion
/// Convert a CXF timestamp to a [DateTime<Utc>].
```